drivers/crypto/Kconfig.rpi_pico

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+config CRYPTO_RPI_PICO_SHA256
+	bool "RasberryPi Pico SHA256 Accelerator driver"
```

```suggestion
	bool "Raspberry Pi Pico SHA256 Accelerator driver"
```

or maybe even like this, as Pico is a board and the driver is for SoC. It's opinionated though, feel free to reject:

```suggestion
	bool "Raspberry Pi microcontrollers SHA256 Accelerator driver"
```